pkg/registry/resource/rest/storage_resource.go

```diff
@@ -50,6 +49,10 @@ func (p RESTStorageProvider) NewRESTStorage(apiResourceConfigSource serverstorag
 func (p RESTStorageProvider) v1alpha3Storage(apiResourceConfigSource serverstorage.APIResourceConfigSource, restOptionsGetter generic.RESTOptionsGetter) (map[string]rest.Storage, error) {
 	storage := map[string]rest.Storage{}
 
+	// Registered also without the corresponding feature gate because it
+	// might be useful to provide access to these resources while their
+	// feature is off to allow cleaning them up.
+
```

Comment:
What code block is this meant to be attached to -- is there just an extra newline, by mistake?

Reply:
It's a comment about all of the `if` checks below, not one in particular as it was before.

I could also put it into a function comment, if that's less confusing.